### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by chrislovecnm**
* docs/README.md formatting and bug fixes
* docs/manifests_and_customizing_via_api.md formatting fixes

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -28,29 +28,30 @@
 * [kubectl](kubectl.md)
     * how to point kubectl to your `kops` cluster
 
+## Advanced / Detailed List of Configurations
 
-## Operations
+### API / Configuration References
+* [Godocs for Cluster - `clusterSepc`](https://godoc.org/k8s.io/kops/pkg/apis/kops#ClusterSpec).
```
Comment (andrewsykim) [inline docs/README.md]: `s/clusterSepc/ClusterSpec`

File: docs/README.md [hunk 2]
```diff
@@ -28,29 +28,30 @@
...
+* [Godocs for Instance Group - `instanceGroupSpec`](https://godoc.org/k8s.io/kops/pkg/apis/kops#InstanceGroupSpec).
```
Comment (andrewsykim) [inline docs/README.md]: `s/instanceGroupSpec/InstanceGroupSpec`

File: docs/README.md [hunk 3]
```diff
@@ -28,29 +28,30 @@
...
+
+### API Usage Guides
+* [`kops` cluster API definitions](cluster_spec.md)
+    * overview of some of the API value to customize a `kops` cluster
+* [`kops` instance groups API](instance_groups.md)
+    * overview of some of the API value to customize a `kops` groups of ks8 nodes
```
Comment (andrewsykim) [inline docs/README.md]: `s/ks8/k8s`

File: docs/README.md [hunk 4]
```diff
@@ -28,29 +28,30 @@
...
+* [Using Manifests and Customizating via the API](manifests_and_customizing_via_api.md)
```
Comment (andrewsykim) [inline docs/README.md]: `s/Customizating/Customizing`

**Comment by justinsb** [comment]
LGTM after typos - please self-merge after fixing

**Pull request merged**